ReleaseHistory.md

```diff
@@ -16,6 +16,7 @@
 - NEW => new feature 
 
 ## UNRELEASED
```

Comment:
>UNRELEASED [](http://example.com/codeflow?start=3&length=10)

can we go ahead and burn a 4.2.1 release next? Seems like this issue is blocking lots of people.

Reply:
@suvamM if sounds good to you I can start this.

Reply:
Release is finished.